bin/quicknxs2

```diff
@@ -36,14 +36,6 @@ sys.excepthook = no_abort_excepthook
 for item in sys.path:
     if '.local' in item and 'reflectivity_ui' not in item:
         sys.path.remove(item)
-    for item in sys.modules:
```

Comment:
Why is this blocks of codes removed?

Reply:
Because `logging.exception()` does the whole everything that was there before in a single line